### ISSUE TITLE
Not detecting external named links

### ISSUE THREAD

**Issue by behnam** [rlaboiss/pelican-linkclass#1]
For example, this markdown:
```markdown
This is an [external link][my_url].

[my_url]: https://github.com/rlaboiss/pelican-linkclass/
```

The URL is external, but it doesn't get tagged as such.

**Comment by rlaboiss** [comment]
The linkclass extension is intended to work only with Markdown inline links. I will take a look at extending it to reference-style links.

**Comment by rlaboiss** [comment]
Fixed in fdc81a84b78d331fadeaeeb301436e171e2247b4

**Comment by behnam** [comment]
Awesome! Thanks, @rlaboiss!
